.github/workflows/pipeline.yml

```diff
@@ -29,11 +29,16 @@ jobs:
           cd ../backend
           npm ci
 
+      - name: Create env file
+        run: |
+          touch ./backend/.env
+          echo ADMIN_EMAIL_SENDER_PASSWORD=${{ secrets.ADMIN_EMAIL_SENDER_PASSWORD }} > ./backend/.env
```

Comment:
how many hours will this save?

Reply:
this exact line of code added 5 hours to my time spent debugging env files on pipeline XDXDXDXDXDXD